### ISSUE TITLE
Setup isn't easy for non Pythonistas

### ISSUE THREAD

**Issue by sayopaul** [HAKSOAT/WebScraping#1]
Hey there Ali . You can improve your readme by stating the instructions clearly . Someone who hasn't been in the Python world would not realize that they need to use pip to install the requirements before getting it to work . :grimacing:

**Comment by HAKSOAT** [comment]
Ok. I'd work on that.

**Comment by deven96** [comment]
So what would this be, a GUI that outputs csv files?

**Comment by HAKSOAT** [comment]
Nah. He was just referring to finding it hard to install the requirements.
